jans-cli-tui/cli_tui/plugins/999_jans/main.py

```
@@ -106,3 +129,198 @@ async def coroutine():
+        create_client_button_label = _("Create Client")
+        create_client_button = Button(create_client_button_label, handler=self.create_ssa_cli, width=len(create_client_button_label)+4)
+        create_client_button.keep_dialog = True
+        buttons = [
+            create_client_button,
+            Button(_("Cancel"))
+        ]
+        mydialog = JansGDialog(self.app, title=_("SSA for Creating Client"), body= body, buttons=buttons)
+        self.app.show_jans_dialog(mydialog)
+
+    def clear_config(self):
+        def do_clear_config(dialog):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `do_clear_config`

Add return type annotation: `None`

(ANN202)